CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 * **[Fix]** Check for running in App Center Test will now work when using AndroidX instead of the support library.
 * **[Feature]** Add `AppCenter.isRunningInAppCenterTestCloud` to provide method to check if the application is running in Test Cloud.
 
+### App Center Data
+
+* **[Feature]** Change the default offline persistence from 1 day to infinite.
```

Comment:
Let's use the same wording as https://github.com/microsoft/appcenter-sdk-apple/pull/1678 for consistency.